apps/mobile/src/screens/ChatRoomWebView/ChatRoomWebView.js

```diff
@@ -4,7 +4,7 @@ import useRouteParams from 'hooks/useRouteParams'
 import HyloWebView from 'components/HyloWebView'
 import KeyboardFriendlyView from 'components/KeyboardFriendlyView'
 
-export const DEFAULT_CHAT_TOPIC = 'home'
+export const DEFAULT_CHAT_TOPIC = 'general'
```

Comment:
oy, how is this broken now?

Reply:
This is a backup, so that this view always has a topic... I guess it is kinda broken in prod now, since its probably defaulting to `home`. But ideally its never relied on